cet/atlas/mongosync-compatibility-restriction.rst

```diff
@@ -0,0 +1,9 @@
+.. important:: 
+
+   If you are using {+dedicated-clusters+} for your deployment and plan to use 
+   Live Migration in the {+atlas-ui+} or standalone |mongosync|, 
+   do NOT choose :guilabel:`Latest Version With Auto Upgrades`. 
+   This option auto upgrades your {+cluster+} to the latest minor release. 
+   Some minor releases, such as MongoDB version 8.2, may not be supported by Live 
+   Migration or Mongosync. Choose a major version instead to ensure compatibility 
```

Comment:
When upgrading, choose a major version to upgrade to instead. This ensures compatibility with Live Migration and mongosync. 

JM: updrage -- this should be a link. Find a link to the docs on upgrading and make the word upgrade use that link. 

```
[upgrade
](https://www.mongodb.com/docs/atlas/tutorial/major-version-change/#upgrade-your-major-mongodb-version)
```

Also: Live Migration and mongosync are not products and we should not call them such. 

Also, we had a note already and could have used the text from that note. That text was clear and reviewed, and I don't see a reason for rewriting that note here, if it was me, I'd just expand the existing note.